src/infra/webapi/Models/User.cs

```diff
@@ -5,7 +5,11 @@ namespace VgInventory.Infra.WebApi.Models
     public class User
     {
         public string Id { get; set; }
+        public string Email { get; set; }
         public string UserName { get; set; }
-        public DateTime CreationDate { get; set; }
+        public DateTime StartDate { get; set; }
+        public long StartDateUnix { get; set; }

```

Comment:
Not sure what this is used for in the frontend. Also StartDate will be a problem, unless either the frontend or backend parses